alf/algorithms/sac_algorithm.py

```diff
@@ -349,14 +349,8 @@ def train_step(self, exp: Experience, state: SacState):
         info = SacInfo(actor=actor_info, critic=critic_info, alpha=alpha_info)
         return PolicyStep(action_distribution, state, info)
 
-    def train_complete(self,
-                       tape: tf.GradientTape,
-                       training_info: TrainingInfo,
-                       weight=1.0):
-        ret = super().train_complete(
-            tape=tape, training_info=training_info, weight=weight)
+    def after_train(self):
```

Comment:
need to implement the following (see Algorithm._trainable_attributes_to_ignore)
```python
def _trainable_attributes_to_ignore(self):
    return ['_target_critic_network1', '_target_critic_network2']
```

Reply:
added